### PR TITLE
OpenQASM introduction page

### LINKED ISSUE #93: [ NEW PAGE ] What is OpenQASM
## What the new page should cover:
Focus on Import/Export & interoperability

## SME collaborators (involve for planning and review):
jake Lishman, Luciano Bello

### PR DESCRIPTION
**Pull request by abbycross**
Closes #93

### CODE REVIEW

**Comment by abbycross** [comment]
cc: @awcross1

File: docs/build/introduction-to-qasm.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Introduction to OpenQASM
+description: An introduction to OpenQASM (Open quantum assembly language)
+---
+
+# Introduction to OpenQASM
+
+OpenQASM (open quantum assembly language), an imperative programming language for describing quantum circuits, is a platform-agnostic (or is "machine-independent" better?) programming interface for IBM Quantum systems. OpenQASM uses the measurement-based quantum circuit model to express quantum programs as ordered sequences of parameterized operations (such as gates, measurements, and resets) and real-time classical computation. OpenQASM can also describe circuits intended to characterize, validate, or debug quantum systems.
```
Comment (abbycross) [inline docs/build/introduction-to-qasm.mdx:8]:
```suggestion
OpenQASM (open quantum assembly language),  a machine-independent programming interface for IBM Quantum systems, is an imperative programming language for describing quantum circuits. OpenQASM uses the quantum circuit model to express quantum programs as ordered sequences of parameterized operations (such as gates, measurements, and resets) and real-time classical computation. In addition to quantum algorithms, OpenQASM can describe circuits intended to characterize, validate, or debug quantum systems.
```

**Comment by abbycross** [comment]
@blakejohnson does the message come across strongly enough about OpenQASM as a bridge among tools with the latest updates I've made? I also added some more "why" with an allusion to the multiple audiences it accommodates: https://qiskit-docs-preview-pr-295.1799mxdls7qz.us-south.codeengine.appdomain.cloud/build/introduction-to-qasm

File: docs/build/introduction-to-qasm.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Introduction to OpenQASM
+description: An introduction to OpenQASM (Open quantum assembly language)
+---
+
+# Introduction to OpenQASM
+
+OpenQASM (open quantum assembly language), a machine-independent programming interface for IBM Quantum systems, is an imperative programming language for describing quantum circuits. OpenQASM uses the quantum circuit model to express quantum programs as ordered sequences of parameterized operations (such as gates, measurements, and resets) and real-time classical computation. In addition to quantum algorithms, OpenQASM can describe circuits intended to characterize, validate, or debug quantum systems.
```
Comment (blakejohnson) [inline docs/build/introduction-to-qasm.mdx:8]: This turn of phrase sounds like OpenQASM is only for IBM systems: "a machine-independent programming interface for IBM Quantum systems". Or at the very least, it sounds like an oxymoron. Rephrase?

**Comment by blakejohnson** [comment]
@abbycross yes, it comes through.

File: docs/build/introduction-to-qasm.mdx (hunk 2 above)
Comment (abbycross) [reply docs/build/introduction-to-qasm.mdx:8]: How about "a machine-independent programming interface compatible with IBM Quantum Systems"?

Comment (blakejohnson) [reply docs/build/introduction-to-qasm.mdx:8]: Yes, that works.

Comment (abbycross) [inline docs/build/introduction-to-qasm.mdx:8]:
```suggestion
OpenQASM (open quantum assembly language), a machine-independent programming interface compatible with IBM Quantum systems, is an imperative programming language for describing quantum circuits. OpenQASM uses the quantum circuit model to express quantum programs as ordered sequences of parameterized operations (such as gates, measurements, and resets) and real-time classical computation. In addition to quantum algorithms, OpenQASM can describe circuits intended to characterize, validate, or debug quantum systems.
```

**Review by blakejohnson — APPROVED** [review]

File: docs/build/introduction-to-qasm.mdx
Comment (javabster) [inline docs/build/introduction-to-qasm.mdx:1]: The only thing I feel is missing from this piece is some code. Would it be possible to include a section that just shows an example of what a typical qasm program looks like and how to interpret the syntax? Doesn't have to be a lot, just something to give readers a flavour and then they can dig into the paper in more detail

**Review by javabster — APPROVED** [review]
LGTM 🚀

**Pull request merged**